src/lib.rs

```diff
@@ -411,18 +439,7 @@ impl TestnetDeployer {
         if !environments.contains(&name.to_string()) {
             return Err(Error::EnvironmentDoesNotExist(name.to_string()));
         }
-
-        self.ansible_runner.run_playbook(
-            AnsiblePlaybook::StartNodes,
-            AnsibleInventoryType::Genesis,
-            None,
-        )?;
-        self.ansible_runner.run_playbook(
-            AnsiblePlaybook::StartNodes,
-            AnsibleInventoryType::Nodes,
-            None,
-        )?;
-
+        self.ansible_provisioner.start_nodes().await?;
```

Comment:
Just a small question, do we also start the bootstrap nodse here?

Reply:
Yeah, it would be useful to have the bootstrap nodes in here too.